common/djangoapps/entitlements/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf.urls import url, patterns, include
+from rest_framework.routers import DefaultRouter
+
+from .views import EntitlementViewSet
+
+router = DefaultRouter()
+router.register(r'entitlements', EntitlementViewSet, base_name='entitlements')
+
+urlpatterns = patterns(
+    'entitlements.views',
+    url(r'^', include(router.urls, namespace='api')),
```

Comment:
No need for the `^`. It's the same as an empty string.

Reply:
If you move this file to `common/djangoapps/entitlements/api/v1/urls.py`, you shouldn't need a `namespace`.

Reply:
Do I move the tests and migrations as well?

Reply:
Migrations stay where they are. Here is my preferred layout:

```
/entitlements
-api
--v1
---tests/
----test_filters.py
----test_serializers.py
----test_views.py
---filters.py
---serializers.py
---urls.py
---views.py
--urls.py
-migrations/
-tests/
--test_models.py
-models.py
-urls.py
```

Reply:
Please take a look at what I changed it to.  Filters and Serializers should be API version specific as well?